disco_aws_automation/disco_aws.py

```diff
@@ -394,6 +394,24 @@ def provision(self, ami, hostclass=None, owner=None, instance_type=None, monitor
 
         elb = self.update_elb(hostclass, update_autoscaling=False, testing=testing)
 
+        tags = {"hostclass": hostclass,
```

Comment:
```suggestion
        tags = {
            "hostclass": hostclass,
```
A bit of a nitpick, but I don't like the whitespacing that this generates when we have the curly bracket and the first item on the same line. I think the above is easier to follow.